src/commands/createNewProject/PowerShellProjectCreator.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { extInstallTaskName, func, funcWatchProblemMatcher, hostStartCommand, ProjectRuntime, TemplateFilter } from "../../constants";
+import { powershellDebugConfig } from "../../debug/PowerShellDebugProvider";
+import { ScriptProjectCreatorBase } from './ScriptProjectCreatorBase';
+
+export class PowerShellProjectCreator extends ScriptProjectCreatorBase {
+    public readonly templateFilter: TemplateFilter = TemplateFilter.All;
```

Comment:
As mentioned already, you can add PowerShell templates to the "verified" category here:
https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/templates/ScriptTemplateRetriever.ts#L119

And this should default to TemplateFilter.Verified

Reply:
Is there any reason that you can think of that the existing templates wouldn't work with PowerShell?

The `function.json`'s are the identical to the JavaScript function templates... the only thing different is... well... the code. But that shouldn't effect whether the template bindings work, right?